src/Components/PaperEditor/TranscriptsContainer/LabelsList/LabelForm.js

```diff
@@ -48,8 +48,9 @@ const LabelForm = (props) => {
     setColor(randomColor());
   };
 
-  const handleColorPickerChangeComplete = () => {
-    setColor({ color: chroma(color.hex).name() });
+  const handleColorPickerChangeComplete = (e) => {
+    const colorValue = e.hex;
+    setColor(chroma.valid(colorValue) ? chroma(colorValue).name() : colorValue);
```

Comment:
I don't understand how `colorValue` can work if it's not valid?

Reply:
Welll... yes. Tbh thinking about it I don't even know why we're checking if it's valid because the colour values are hard-coded as far as I understand. So they will always be valid. 

Anywhoodles though I'll modified this so if for whatever reason a colour blows up, it just chooses a random colour.